### DISCUSSION TITLE
Update extended-scalars and extended-validation to 16.1

### DISCUSSION THREAD

**Discussion by setchy** [graphql-java/graphql-java#2175] [General]
Hi all,  

I had posted this in those repos and on Spectrum, but thought I'd try the new GitHub discussions.

Our organizations move from graphql-java:15 -> graphql-java:16.1 is dependent on updates to **graphql-java-extended-scalars** and **graphql-java-extended-validation**, particularly due to the renaming of ExecutionPath to ResultPath.

**graphql-java-extended-scalars**
[Pull request 33](https://github.com/graphql-java/graphql-java-extended-scalars/pull/33)

**graphql-java-extended-scalars**
[Pull request 39](https://github.com/graphql-java/graphql-java-extended-validation/pull/39) already merged to [master](https://github.com/graphql-java/graphql-java-extended-validation), but I don't have access to do a release

If one of the maintainers could please assist this would be greatly appreciated

**Comment by setchy** [comment]
Both have been released. Thanks @bbakerman

**Reply by bbakerman** [reply]
Wow your right on it :)

**Reply by setchy** [reply]
😁

This system app really helps me keep up to date  https://github.com/manosim/gitify

**Reply by setchy** [reply]
Just saw @andimarek post about 16.2 🤣
